### PR TITLE
docs: add link to The Global and Mail guide to using SecureDrop

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Erin suggested this would help localizers who want to learn about
SecureDrop.

## Testing

N/A

## Deployment

N/A

### If you made changes to documentation:

- [X] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: docs/development/l10n.rst [hunk 1]
```diff
@@ -4,7 +4,8 @@ Getting Started with Weblate Translations
 SecureDrop is a whistleblower submission system for media
 organizations to securely accept documents from and
 communicate with anonymous sources. For more information, you may be
-interested to learn about :doc:`what makes SecureDrop unique
+interested to watch `The Global and Mail guide to using SecureDrop <https://www.youtube.com/watch?v=oSW2wMWtAMM>`_
```
Comment (redshiftzero) [inline docs/development/l10n.rst]: Nit: `Global and Mail` -> `Globe and Mail`

Comment (dachary) [reply docs/development/l10n.rst]: :-) Interesting typo on my part. Good catch!

**Comment by dachary** [comment]
fixed & repushed!

**Comment by dachary** [comment]
@redshiftzero commit is even more effective ;-)

**Pull request merged**